.bazelrc

```diff
@@ -111,6 +113,12 @@ build:coverage --experimental_fetch_all_coverage_outputs
 build:coverage --collect_code_coverage
 build:coverage --instrumentation_filter="^//src/v[/:]"
 
+# =================================
+# Debug Symbols
+# =================================
+build:debug-symbols --copt -g
```

Comment:
Can we remove this now?

Reply:
yeah, sorry. It was a left-over from before.
Done